system/apps_ds213/58_gabmini/build.sh

```diff
@@ -1,3 +1,8 @@
-./build_ds203.sh
-./build_ds213.sh
+if [ "$1" == "ds203" ]; then
+	exec ./build_ds203.sh
+elif [ "$1" == "ds213" ]; then
+	exec ./build_ds213.sh
+fi
```

Comment:
what about other applications that have multiple targets?

Reply:
I didn't go through all of them to add these conditions. I focused mainly on ds213. I'll check all of them.

Reply:
It would be great if these scripts would build for all platforms when no argument is provided

Reply:
Will do. For the moment, since I started going through all the app's `build.sh` I realized it's mostly copy-paste. So now I'm trying to make a 'generic' app builder script in order to minimize code duplication within `build.sh`.